app/forms/field_handlers/field_handler.py

```diff
@@ -43,11 +48,19 @@ def label(self):
     def guidance(self):
         return self.answer_schema.get("guidance", "")
 
+    @staticmethod
+    def format_question_title(error_message, question_title):
+        error_message = error_message or ""
```

Comment:
When can `error_message` not be set?

Reply:
The existing unit tests were not setting the messages. I've updated the tests to either disable validation or use MagicMock where appropriate.